src/components/messaging/chat.header.tsx

```diff
@@ -55,43 +56,46 @@ class ChatHeaderComponent extends React.Component<ChatHeaderProps> {
     );
   };
 
-  private renderProfileAvatar = (): React.ReactElement<AvatarProps> | null => {
+  private renderProfileAvatar = (): React.ReactElement<AvatarProps> => {
     const { interlocutor } = this.props;
 
-    return interlocutor ? (
+    return (
       <Avatar
         source={{ uri: interlocutor.photo }}
         shape='round'
         size='small'
       />
-    ) : null;
+    );
   };
 
-  private renderRightControls = (): React.ReactElement<TopNavigationBarActionProps>[] => {
-    return ([
+  private renderRightControls = (): React.ReactElement<TopNavigationActionProps> => {
+    return (
       <TopNavigationAction
         icon={this.renderProfileAvatar}
         onPress={this.onProfile}
-      />,
-    ]);
+      />
+    );
+  };
+
+  private renderInterlocutorProps = (): TopNavigationProps | null => {
```

Comment:
Render props?

Reply:
Exactly. Because it provides property name and its value